projects/drivers/util/util.c

```diff
@@ -188,3 +190,46 @@ uint64_t div_u64(uint64_t dividend, uint32_t divisor)
 
 	return div_u64_rem(dividend, divisor, &remainder);
 }
+
+/**
+ * Compare two strings
+ * @param *str1 pointer to string 1
+ * @param *str2 pointer to string 2
+ * @return TRUE if strings are equal, 0 otherwise
+ */
+bool strequal(const char *str1, const char *str2)
```

Comment:
I would not create a new function that's only a !strcmp()... but there might be a good reason.

Reply:
"strequal" was in the initial version of the project. "!strcmp()" will work fine, I will change that.